### ISSUE TITLE
Remove ewatercycle.parametersetdb

### ISSUE THREAD

**Issue by sverhoeven** [eWaterCycle/ewatercycle#131]
Possible once #118 is merged.

The `ewatercycle.parametersetdb` module does 3 things:
1. Downloading of datafiles using `svn export`
2. Fetching config file from Internet
3. Editing config file as a dict

Thing 1 and 2 are also handled by [ExampleParameterSet.download()](https://github.com/eWaterCycle/ewatercycle/blob/e946b8e384fbbb38b61074b4f33ad8d97b9808f0/ewatercycle/parameter_sets/_example.py#L31)
Thing 3 is used in the `constructor/setup()/parameters` of the `ewatercycle.models.*` models. 
Pcrglobwb and wflow both use `ewatercycle.pararmeterserdb.config.CaseConfigParser`.
So thing 1 and 2 are no longer needed and having thing 3 in parametersetdb is weird as its needed in the models not in a parameter set.

My suggestion would be 
1. to move `ewatercycle.pararmeterserdb.config.CaseConfigParser` to `ewatercycle.util.CaseConfigParser` 
2. remove the `ewatercycle.parametersetdb`
3. remove any tests on `ewatercycle.parametersetdb`
4. Update all notebooks in repo to no longer use `ewatercycle.parametersetdb`

**Comment by Peter9192** [comment]
Good idea 👍 

Note that AbstractConfig is also used for Lisflood:

https://github.com/eWaterCycle/ewatercycle/blob/8d1c3077c366260b171d9f0880335f11c3c30d57/ewatercycle/models/lisflood.py#L331-L332
